src/libostree/ostree-repo-pull.c

```diff
@@ -812,21 +821,40 @@ meta_fetch_on_complete (GObject           *object,
   gboolean free_fetch_data = FALSE;
 
   ostree_object_name_deserialize (fetch_data->object, &checksum, &objtype);
-  checksum_obj = ostree_object_to_string (checksum, objtype);
-  g_debug ("fetch of %s%s complete", checksum_obj,
-           fetch_data->is_detached_meta ? " (detached)" : "");
+  g_debug ("fetch of %s%s%s%s complete", ostree_object_to_string (checksum, objtype),
+           (fetch_data->type == OSTREE_FETCH_OBJECT_DETACHED_METADATA) ? " (detached)" : "",
+           (fetch_data->type == OSTREE_FETCH_OBJECT_COMPAT_SIZES) ? " (compat sizes)" : "",
+           (fetch_data->type == OSTREE_FETCH_OBJECT_COMPAT_SIGNATURE) ? " (compat signature)" : "");
```

Comment:
Nitpicky, but here you've dropped the use of the `checksum_obj` temporary variable since the code I had still uses `ostree_object_to_string` directly. Probably better to use the new variable from upstream, but if not, then the declaration for `checkum_obj` should be removed since it's no longer used.